h1.htm

```diff
@@ -0,0 +1,171 @@
+<caption style="text-align:left;"><h2><b>About<b></h2></caption><hr>
+<p >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+</div>
+<div>
+     <div class="container div1">
+        <img class="pic" src="John Doe.jpeg" alt="John Doe" > 
+        <h3>John Doe</h3>
+        <p>CEO & Founder<br><br>Phasellus eget enim eu lectus faucibus vestibulum.<br>Suspendisse sodales pellentesque elementum.</p> 
+        <p><button class="web_button">Contact</button></p>
+     </div>
+     <div class="container div1">
+        <img class="pic" src="Jane Doe.jpeg" alt="Jane Doe" > 
+        <h3>Jane Doe</h3>
+        <p>Architect<br><br>Phasellus eget enim eu lectus faucibus vestibulum.<br>Suspendisse sodales pellentesque elementum.</p> 
+       <button class="web_button">Contact</button>
+   </div>
+    <div class="container div1">
+       <img class="pic" src="Mike Ross.jpeg" alt="Mike Ross" >
+       <h3>Mike Ross</h3>
+       <p>Architect<br><br>Phasellus eget enim eu lectus faucibus vestibulum.<br>Suspendisse sodales pellentesque elementum.</p> 
+       <button class="web_button">Contact</button>
+   </div>
+    <div class="container div1">
+      <img class="pic" src="Dan Star.jpeg" alt="Dan Star" > 
+      <h3>Dan Star</h3>
+      <p>Architect<br><br>Phasellus eget enim eu lectus faucibus vestibulum.<br>Suspendisse sodales pellentesque elementum.</p> 
+      <button class="web_button">Contact</button><br><br>
+    </div>
+
+</div>
+
+
+                <!--THIS IS FOR FORM-->
+
+
+<div>
+<caption style="text-align:left;"><h2><b>Contact<b></h2></caption>
+<hr>
+<p></h3>Let's get in touch and talk about your next project</h3></p>
+
+<form>
+    <label for="name">Name:</label><br>
+    <input style="width:100%;" type="text" id="name"><br>
+
+    <label for="Email">Email ID:</label><br>
+    <input style="width:100%;" type="text" id="EMail"><br>
+
+    <label for="number">Mobile:</label><br>
+    <input style="width:100%;" type="text" id="number"><br>
+
+    <label for="subject">Subject:</label><br>
+   <input style="width:100%;" type="text" id="subject"><br>
+
+   <label for="comment">Comment:</label><br>
+   <input  style="width:100%;" type="text" id="comment"><br><br>
+
+   <input style="width:15%; padding:10px;border:20px; margin:10px; background-color:black; color:white;" type="submit" value="Send Message"><br><br>
+
+</form>
+</div>
+
+               <!--THIS IS FOR MAP PICTURE-->
+<div>
+<img src="map.jpeg",alt="glass building",width="150px",height="80px".
```

Comment:
Its missing the Image File , Please push and ask for review.

Reply:
Added all the required image files.